Sources/Gen/Gen.swift

```diff
@@ -13,6 +13,7 @@ public struct Gen<Value> {
   /// - Parameter rng: A random number generator.
   /// - Returns: A random value.
   @inlinable
+  @Sendable
```

Comment:
That’s good feedback. I didn’t include those annotations at first, but the compiler highlighted an issue with composability of the methods. I’m on vacation at the moment so I don’t have my mac in front of me, but I remember there were a couple of times in the library code where a function was passed as a parameter and needed to be Sendable. Looks like ‘shuffled’ was one of them. Rather than just annotate a couple methods I included the annotation on all similar methods. I could change those cases to use an explicit closure and call the method from inside the body if you prefer. I just didn’t see a downside to including the annotation, especially since it was used in the library itself.

Reply:
@seanmrich I think those instances could potentially be refactored, but it'd help if you could share whenever you're back, no rush :smile: